feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt

```diff
@@ -84,8 +88,12 @@ private fun SearchScreen(
         modifier = modifier,
         topBar = {},
         content = { innerPadding ->
+            val insets = WindowInsets(
+                top = innerPadding.calculateTopPadding(),
+                bottom = innerPadding.calculateBottomPadding()
+            ).union(WindowInsets.ime)
             Column(
-                modifier = Modifier.padding(innerPadding)
+                modifier = Modifier.windowInsetsPadding(insets)
```

Comment:
Thanks to you, I found `safeDrawingPadding()` method of Modifier. can we use it?

Reply:
Oh, Probably we should apply only the top and bottom.
Can you use the lambda version of this function? Because it will use remember in it.

```kotlin
windowInsetsPadding(
    noinline inspectorInfo: InspectorInfo.() -> Unit,
    crossinline insetsCalculation: WindowInsetsHolder.() -> WindowInsets
)
```

Reply:
Sorry, this function is a private function.
So, how about using `remember`? Sorry for keeping changing my opinions

Reply:
Sorry, I leaned a lot of things 😅 Can you use `WindowInsets.safeDrawing.only(WindowInsetsSides.Vertical)`
https://github.com/DroidKaigi/conference-app-2022/pull/463/files#diff-506394b5d18971e4dcc8f5857b755c959c04eafd90c331702cf53a65539fc17dR413